### PR TITLE
Final touches

### PR DESCRIPTION
**Pull request by patmessina**
Renamed package and updated Readme. 

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,96 +1,122 @@
...
-Add `zgen load dbz/kube-aliases` to your zgen plugins in your `.zshrc`
+```yaml
+cmd:
+  - short: g
+    cmd: get
+```
 
-### Bash
+Will generate all get commands for specified resource, i.e. `alias kgp="kubectl get pods"`
+Prefix and suffix will generate commands before and after the alias respectively.
```
[the remark this answers is not in the document]
Comment (patmessina) [reply README.md:88]: Sure

File: README.md [hunk 2]
```diff
@@ -1,96 +1,122 @@
...
-alias kns='kubens'
+Any other aliases can be generated by the following
+
+```yaml
+additional:
+  - short <short>
+    cmd: <cmd>
+```
+
+The following will generate `alias eh="echo hello"`
```
[the remark this answers is not in the document]
Comment (patmessina) [reply README.md:102]: Can do

File: README.md [hunk 3]
```diff
@@ -1,96 +1,122 @@
...
+
+```yaml
+additional:
+  - short eh
+    cmd: "echo hello"
 ```
 
-# Customizing Aliases
+# Installation
```
[the remark this answers is not in the document]
Comment (patmessina) [reply README.md:110]: Fine by me.

File: README.md [hunk 4]
```diff
@@ -1,96 +1,122 @@
...
+
+To simply grab a bunch of aliases without customization simply get the
+`aliases` file and source it in your `bashrc` or `zshrc` file.
+
+## Generate Aliases
```
[the remark this answers is not in the document]
Comment (patmessina) [reply README.md:115]: Yea, it should be.

File: README.md [hunk 5]
```diff
@@ -1,96 +1,122 @@
...
-kfind      # use a regular expression to find items across everything except
-           # custom resources
-kgpns      # Get just pod names in a namespace
-kl         # kubectl logs <podname>
-klf        # kubectl logs -f <podname>: i.e. watch logs live
-kpfind     # Search through pods with regular expressions
-kdfind     # Search through deployments with regular expressions
-krd        # restart a deployment
-kstatus    # search across namespaces to find pods statuses
+generate-kube-aliases aliases.yaml ~/.aliases
```
[the remark this answers is not in the document]
Comment (patmessina) [reply README.md:29]: Ill expand on this.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (patmessina) [reply README.md]: https://github.com/Dbz/kube-aliases/pull/30/commits/c98c3d0652f0311bb7ead5d75b49942da290ee09

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (patmessina) [reply README.md:176]: https://github.com/Dbz/kube-aliases/pull/30/commits/726b2cca878bf75aa3ce17895ac79032cc0e4071 

Like this?

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (patmessina) [reply README.md:159]: https://github.com/Dbz/kube-aliases/pull/30/commits/19e80f98528ad266f312fea3df2dbf6554c94aa3

File: README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (patmessina) [reply README.md]: I did so here https://github.com/Dbz/kube-aliases/pull/30/commits/16ff616c6f2398c4febb07b234dfd1ea1ea0bd00

What do you think?

File: README.md [hunk 6]
```diff
@@ -1,96 +1,230 @@
...
+kdel<r>     # kubectl delete <resource>, e.g. kdelp for kubectl delete pods
+kd<r>       # kubectl describe <resource>, e.g. kdp for kubectl describe pod
+ke<r>       # kubectl edit <resource>, e.g. kep for kubectl edit pods
+kg<r>       # kubectl get <resource>, e.g. kgp for kubectl get pods
+kga<r>      # kubectl get --all-namespaces -o wide <resource>, e.g. kgap for kubectl --all-namespaces -o wide get pods
+```
+
+## Generating Aliases
+
+Want to generate custom aliases? [Install](#generate-aliases) the latest `generate-kube-aliases`
```
Comment (Dbz) [inline README.md:40]: I feel like this should link to the releases page

File: README.md [hunk 7]
```diff
@@ -1,96 +1,230 @@
...
-ke<r>       # kubectl edit <resource>, e.g. kgp for kubectl edit pods
-kg<r>       # kubectl get <resource>, e.g. kgp for kubectl get pods
-kga<r>      # kubectl get --all-namespaces -o wide <resource>, e.g. kgap for kubectl --all-namespaces -o wide get pods
+echo "source ${HOME}/.kube-aliases" >> .bashrc
 ```
 
-However, these aliases can be customize by editing the config file
-aliases.yaml. See customizing.
+Or following the same steps, one can create a smaller list of only additional
+content. For example in a `additonal.yaml` file
```
Comment (Dbz) [inline README.md:59]:
```suggestion
content. For example in an `additonal.yaml` file
```

File: README.md [hunk 8]
```diff
@@ -1,96 +1,230 @@
...
+  - short: g
+    cmd: get
+```
+
+Will generate aliases for get commands for each specified resource in the
+format `alias kg<resource>=kubectl get <resource>`.  An example with the `pod`
+resource is `alias kgp=kubectl get pod`.  Prefix and suffix will generate
+commands before and after the alias respectively.
+
+Prefix example that will generate `alias wkg<resource short>=watch kubectl get <resource>`.
```
Comment (Dbz) [inline README.md:166]:
```suggestion
Here is an example cmd prefix that will generate `alias wkg<resource short>=watch kubectl get <resource>`:
```

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Dbz) [reply README.md:187]: Give that you added the example for `alias krd="kubectl rollout restart deployment"`, I think we delete this example and just rely on the following krd example

File: README.md [hunk 9]
```diff
@@ -1,96 +1,230 @@
...
+
+```yaml
+additional:
+  - short: krd
+    cmd: "kubectl rollout restart deployment"
+```
+
+# Installation
+
+To simply grab a bunch of aliases without customization simply get the
```
Comment (Dbz) [inline README.md:206]: I'd change:
```
To simply grab a bunch of aliases without customization simply get the
`aliases` file and source it in your `bashrc` or `zshrc` file.
```

To be:
```
To install, download the `aliases` file and source it in your `bashrc` or `zshrc` file.

File: README.md (hunk 6 above)
Comment (patmessina) [reply README.md:40]: I'm planning on adding an Arch Linux package that they could install or if we wanted to add a brew install. Also was planning on adding a regular old curl. Not sure what you would prefer if we add those sections. Link to release or the installation options.

File: README.md (hunk 9 above)
Comment (patmessina) [reply README.md:206]: Looks fine to me.

File: README.md (hunk 6 above)
Comment (Dbz) [reply README.md:40]: Probably both

**Pull request merged**